mmdet/core/bbox/assigners/mask_hungarian_assigner.py

```diff
@@ -70,15 +70,15 @@ def assign(self,
         """
         assert gt_bboxes_ignore is None, \
             'Only case when gt_bboxes_ignore is None is supported.'
-        num_gt, num_query = gt_labels.shape[0], cls_pred.shape[0]
+        num_gt, num_query = gt_labels.shape[0], mask_pred.shape[0]
 
         # 1. assign -1 by default
-        assigned_gt_inds = cls_pred.new_full((num_query, ),
+        assigned_gt_inds = mask_pred.new_full((num_query, ),
+                                              -1,
+                                              dtype=torch.long)
+        assigned_labels = mask_pred.new_full((num_query, ),
```

Comment:
Is this change necessary?

Reply:
K-Net sometimes passes cls_pred=None to this assigner.
We should leave comments here to indicate that why we use the shape of mask_pred